.travis.yml

```diff
@@ -5,23 +5,34 @@ os:
   - macos
   - linux
 elixir:
-  - 1.6
-  - 1.7
-  - 1.8
-  - 1.9
+  - '1.6'
+  - '1.7'
+  - '1.8'
+  - '1.9'
+  - '1.10'
```

Comment:
`1.10` is different from `'1.10'` (notice the single quote mark `'`).

Reply:
I'm not sure I understand what the difference is supposed to be when quoted vs not quoted. Can you provide a bit more detail on the purpose of that change?

Reply:
`1.10` without quote will be parsed as a float number and turned into `1.1`. This was the reason why [this travis build](https://travis-ci.org/bitwalker/timex/builds/649746838?utm_medium=notification&utm_source=github_status) was using Elixir `1.1`.  :rofl: